scripts/gitlab/validate-chainspecs.sh

```diff
@@ -15,6 +15,15 @@ done
 for spec in ethcore/res/ethereum/*.json; do
     if ! ./target/release/chainspec "$spec"; then ERR=1; fi
 done
+
+echo "________Mainnet contains Istanbul EIPs________"
```

Comment:
Doesn't have to be addressed in this PR but would be good to run this on the other testing network (hard forks) such as Contantinople and so on.

In order to make sure that we are not breaking anything.

Reply:
good point, but I think since we're running tests with `istanbul_test` spec, it should be caught there